### DISCUSSION TITLE
Bluetooth control

### DISCUSSION THREAD

**Discussion by Photosyny68** [PaulWoitaschek/Voice#1768] [Ideas]
Option to disable Bluetooth controls
So I won't accidentally hit the buttons

**Comment by PaulWoitaschek** [comment]
You mean the system controls? I don't think that's that an app on the userspace can do. Have you seen this somewhere else?

**Reply by Photosyny68** [reply]
Poweramp has a option to enable/disable Bluetooth controls

**Reply by PaulWoitaschek** [reply]
Can you attach a screenshot of that?

**Comment by Photosyny68** [comment]
![Screenshot_20230406-015634](https://user-images.githubusercontent.com/130021042/230284342-ef08bccd-6292-4365-a095-81a42d54b442.png)
